src/Containers/SparseAxisArray.jl

```diff
@@ -114,6 +114,9 @@ Base.IndexStyle(::Type{<:SparseAxisArray}) = IndexAnyCartesian()
 # eachindex redirect to keys
 Base.keys(::IndexAnyCartesian, d::SparseAxisArray) = keys(d)
```

Comment:
What was the previous behavior of `keys` ? This line redirect to `keys` so if `keys` does not work then according to the comment above, `eachindex` does not work either ?